icon.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Icon loading plugin.
+ *
+ * This plugins loads SVG files and merges them into one SVG sprite
+ *
+ * @example:
+ *      To load the icon `myicons/myicon.svg`:
+ *      ```
+ *      require(["requirejs-dplugins/icon!myicons/myicon.svg"], function (){
+ *          // myicon was added to the sprite
+ *      });
+ *      ```
+ * @module requirejs-dplugins/icon
+ */
+
+define([
+	"./has",
+	"module",
+	"./icon/common"
+], function (has, module, common) {
+	"use strict";
+
+	var cache = common.cache, // paths of loaded icons
+		icons = common.icons; // markup of loaded icons
+
+
+	var loadIcon = {
+		id: module.id,
+
+		/*jshint maxcomplexity: 11*/
+		/**
+		 * Loads an icon file file.
+		 * @param {string} args - The css file to load.
```

Comment:
Also maybe you can find something clearer than args to name the parameter (like icon or iconPath) ?